pom.xml

```diff
@@ -19,16 +19,14 @@
 		<junit.platform.version>1.5.2</junit.platform.version>
 		<hamcrest.version>2.2</hamcrest.version>
 		<maven-jar-plugin.version>3.1.2</maven-jar-plugin.version>
-		<maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+		<maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
```

Comment:
you mentioned you were using the "milestone" version in the operator project as well, are we sure that using this "old" version doesn't break anything?

Reply:
I have tried with `3.0.0-M5` but there is some dependency, which is missing. Anyway, in the next PR I can investigate the problem and change it to this `3.0.0-M5` but for now, we can let it on stable `2.22.2`.

Reply:
These dependencies were needed for that M5 version